pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import { showModal } from './Modal';
+
+import './KeyboardHelpDialog.css';
+
+let keyboardHelpOpen = false;
+
+export function showKeyboardHelpDialog() {
+  if (keyboardHelpOpen) {
+    return;
+  }
+
+  keyboardHelpOpen = true;
+  showModal({
+    title:
+      <span>
+        Keyboard Help (press <kbd>F1</kbd> to open;
```

Comment:
I definitely plan to have a way to open the dialog from somewhere else — I don't want it to be only accidentally discovered by pressing `F1`. This is why I believe it is important to have this text. Also, this dialog might be the first time the user sees any mention of keyboard shortcuts, so I think it is the right time to educate them that they can use `Esc` to close this dialog (and give a hint that this also works in other popups). I wouldn't rely on "Esc being universally understood", especially when it comes to web apps.